### PR TITLE
Schedule component updated

### PR DESCRIPTION
**Pull request by mzachar6**
Remark::
I didn't use Date.parse but put date string directly to new Date()
Please use component method getDateOption(date) as reference

### CODE REVIEW

File: src/pages/Schedule.js [hunk 1]
```diff
@@ -1,11 +1,58 @@
 import React, { Component } from 'react';
+import provider from "../dataProvider";
 
 export default class Schedule extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dates: this.fetchListOfPlayingDates() ? this.fetchListOfPlayingDates() : []
```
Comment (buksy90) [inline src/pages/Schedule.js]: this is ineffective, you are making same network requests twice, moreover, you just throw away result from the first call. In overall, we should try to make as few network requests as possible. Users will also visit this site from their smartphones, and you usually pay for every single transferred megabyte when not on wifi.

also, you are comparing result of fetchListOfPlayingDates  function, but that function doesnt return any data, so theres nothig to compare.

This data will not change, so you can set state.dates to an empty array in constructor and call your fetchListOfPlayingDates function.

Comment (mzachar6) [reply src/pages/Schedule.js]: Thanks for insights and explanation. I've fixed it

File: src/pages/Schedule.js [hunk 2]
```diff
@@ -1,11 +1,58 @@
...
+  }
+
+  getDateOption(date) {
+    const days = ['Nedeľa', 'Pondelok', 'Utorok', 'Streda', 'Štvrtok', 'Piatok', 'Sobota'];
+    const actualDate = new Date(date);
+    const getDay = actualDate.getDay();
+    const getDate = actualDate.getDate();
+    const getMonth = actualDate.getMonth() + 1;
+
+    return `${getDate}.${getMonth} ${days[getDay]}`;
```
[the remark this answers is not in the document]
Comment (mzachar6) [reply src/pages/Schedule.js]: I've created an object Utils which would be container for such a functions in src/utils.js

File: src/pages/Schedule.js [hunk 3]
```diff
@@ -17,17 +20,8 @@ export default class Schedule extends Component {
     })
```
Comment (buksy90) [inline src/pages/Schedule.js]: I just realized ... why is there arrow function in setState? it is unnecessary, you can just pass an object to setState fn

**Comment by mzachar6** [comment]
Could you please review if everything is fine ?

**Comment by buksy90** [comment]
yes, its okay ... you can merge now

**Pull request merged**